### DISCUSSION TITLE
take over pkg?

### DISCUSSION THREAD

**Discussion by sckott** [ropensci/nodbi#38] [General]
@rfhb Any interest in taking over the package maintainer role? I'm leaving rOpenSci, and can't maintain this any longer.

**Comment by rfhb** [comment]
@sckott  Sure, with pleasure & thanks! In fact have worked on it during the last weeks. Will I have to engage with rOpenSci or extract the pkg from the organisation? Hope I can well continue your great work. Thanks again & BW

**Reply by sckott** [reply]
Great news! Thanks much. 

You can engage with the community or not, up to you. You can move the repo to your own account if you like. I'll add you as an admin now!

**Reply by rfhb** [reply]
Alright, will read up on the community and see how to contribute. Best wishes!

**Reply by sckott** [reply]
Sounds good!

**Comment by rfhb** [comment]
Hi @sckott: Please could you let CRAN-submissions@R-project.org know about the maintainer changes? If you could copy me (email address is here: https://github.com/ropensci/nodbi/blob/master/DESCRIPTION), I can refer to this in cran-comments.md for a new submission. Many thanks!

**Reply by sckott** [reply]
Done!

**Reply by rfhb** [reply]
Thanks very much!
